orttraining/orttraining/python/training/execution_agent.py

```diff
@@ -77,9 +77,17 @@ class TrainingAgent(object):
     This is the main class used to run an ORTModule model training.
     """
 
-    def __init__(self, path_or_bytes, session_options=None, providers=None, provider_options=None):
+    def __init__(self, path_or_bytes, fw_feed_names, fw_fetches_names, fw_outputs_device_info,
+                 bw_feed_names, bw_fetches_names, bw_outputs_device_info, session_options=None,
+                 providers=None, provider_options=None):
         """
         :param path_or_bytes: filename or serialized ONNX or ORT format model in a byte string
+        :param fw_feed_names: Feed names for foward pass.
+        :param fw_fetches_names: Fetch names for forward pass.
+        :param fw_outputs_device_info: Device info for fetches in forward pass.
```

Comment:
seems some places we use feed/fetch while other places we use input/output. Should we unify them to input/output?

Reply:
It might make more sense to keep as feeds/fetches since that would keep things really consistent as it would be the same on C++ and Python side. The inconsistency you have pointed out is not really in the name but in the explanation where I say feeds means inputs and fetches means output, this I believe is ok to help users understand the meaning.